Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: Drop subplot-driven tests

subplot is not available in debian, and it would require a lot
of additional packaging work.  Better to just avoid it for now.

This set of changes for the sake of reduced dependency subtrees
was guided by Lars Wirzenius from upstream.

This patched was modified by Peter Michael Green to support
sequoia-sq 0.31.0

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -90,2 +90,0 @@
-[dev-dependencies.subplotlib]
-version = "0.7.0"
@@ -116,3 +114,0 @@
-[build-dependencies.subplot-build]
-version = "0.7.0"
-optional = true
@@ -150,1 +150,0 @@
-subplot = ["subplot-build"]
--- a/build.rs
+++ b/build.rs
@@ -17,5 +17,0 @@
-    // Generate subplot tests.
-    #[cfg(feature = "subplot")]
-    subplot_build::codegen(Path::new("sq.subplot"))
-        .expect("failed to generate code with Subplot");
-
